taskman.yml.dist

```diff
@@ -0,0 +1,35 @@
+github:
+  generate:
+    token: [create a file named taskman.yml and set the token in it.]
+  owner: [The repository owner, normally the company name.]
+  repo: [The repository name.]
+  release:
+    branch: 'master'
+  exclude:
+    labels: [feature-branch, bugfix-branch]
+command:
+  githubchangelog:generate:
```

Comment:
The name of the command is weird... maybe something shorter would be better no ?